app/models/course.rb

```diff
@@ -156,6 +156,10 @@ def self.unsubmitted
 
   scope :ready_for_update, -> { current.or(where(needs_update: true)) }
 
+  scope :ready_for_short_update, lambda {
```

Comment:
I created a new scope, and in this case I just included editathons, set as needs_update true. As the editathons normally last less than one month, current was (as in ready_for_update) is not needed.